src/traccuracy/_tracking_graph.py

```diff
@@ -49,6 +49,8 @@ class NodeFlag(str, enum.Enum):
     FALSE_POS = "is_fp"
     FALSE_NEG = "is_fn"
 
+    MIN_BUFFER_CORRECT = "min_buffer_correct"
```

Comment:
I realized adding this here also means we keep a set for it in the `TrackingGraph`. We don't populate the set or ever get it, but do we want to make a separate enum for flags we don't track? I could also explicitly exclude it from the set creation but that seems messier

Reply:
exclude it from the built sets and add a comment that in future we might want to add a `NodeCounts` or something enum that tracks flags with counts